pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -312,6 +312,7 @@ def __init__(
             )
         self._config_initialized = False
         deepspeed.utils.logging.logger.setLevel(logging_level)
+        pl._logger.setLevel(logging_level)
```

Comment:
@carmocca not sure how I feel about this, setting the global logger level is fine? I liked setting just the deepspeed logger level as it was unintrusive to other messages

Reply:
Wouldn't it make sense that if the user asks to configure the deepspeed logger, we configure ours too? Us checking the current loglevel is ugly and should rely on the logging internals.

What's missing is resetting the log level (hence failing tests)